Dockerfile.ui-monitoring

```diff
@@ -49,7 +49,7 @@ ENTRYPOINT ["/opt/app-root/plugin-backend", "-static-path", "/opt/app-root/web/d
 
 LABEL com.redhat.component="coo-monitoring-console-plugin" \
       name="openshift/monitoring-console-plugin" \
-      version="v0.3.0" \
+      version="v1.0.0" \
```

Comment:
Updated this based on the [VERSION](https://github.com/openshift/monitoring-plugin/blob/main/VERSION). However, please note that other plugins don't seem to have a similar file, but instead rely on tags, which reflect their current `version` labels in this repository, so no changes were made for them.

Reply:
that file is old

Reply:
We track `release-0.3`: https://github.com/rhobs/konflux-coo/blob/25b810a1490058ab6217a210c881ddfccb641ead/.gitmodules#L23

Reply:
Makes sense.